app/main.development.js

```diff
@@ -0,0 +1,338 @@
+import { app, BrowserWindow, Menu, shell, ipcMain } from 'electron';
+import open from 'open';
+import { autoUpdater } from 'electron-updater';
+import { sync as syncActions } from './actions';
+
+let menu;
+let template;
+let mainWindow = null;
+
+if (process.env.NODE_ENV === 'production') {
+  const sourceMapSupport = require('source-map-support'); // eslint-disable-line
+  sourceMapSupport.install();
+}
+
+if (process.env.NODE_ENV === 'development') {
+  require('electron-debug')(); // eslint-disable-line global-require
+  const path = require('path'); // eslint-disable-line
+  const p = path.join(__dirname, '..', 'app', 'node_modules'); // eslint-disable-line
+  require('module').globalPaths.push(p); // eslint-disable-line
+}
+
+app.on('window-all-closed', () => {
+  if (process.platform !== 'darwin') app.quit();
+});
+
+
+const installExtensions = async () => {
+  if (process.env.NODE_ENV === 'development') {
+    const installer = require('electron-devtools-installer'); // eslint-disable-line global-require
+
+    const extensions = [
+      'REACT_DEVELOPER_TOOLS',
+      'REDUX_DEVTOOLS'
+    ];
+
+    const forceDownload = !!process.env.UPGRADE_EXTENSIONS;
+
+    // TODO: Use async interation statement.
+    //       Waiting on https://github.com/tc39/proposal-async-iteration
+    //       Promises will fail silently, which isn't what we want in development
+    return Promise
+      .all(extensions.map(name => installer.default(installer[name], forceDownload)))
+      .catch(console.log);
+  }
+};
+
+app.on('ready', async () => {
+  await installExtensions();
+  const resizable = (process.env.NODE_ENV === 'development' || process.env.BUILD === 'dev');
+
+  mainWindow = new BrowserWindow({
+    show: false,
+    width: 663,
+    height: 769,
+    resizable: resizable
+  });
+
+  mainWindow.loadURL(`file://${__dirname}/app.html`);
+
+  mainWindow.webContents.on('did-finish-load', () => {
+    mainWindow.show();
+    mainWindow.focus();
+    autoUpdater.checkForUpdates();
+  });
+
+  mainWindow.webContents.on('new-window', function(event, url){
+    event.preventDefault();
+    open(url, 'chrome', function(error){
+      if(error){
+        // couldn't open in chrome specifically, probably because it's not installed
```

Comment:
This is actually significantly more complex than I originally thought. The `open` command here isn't sufficient to find and launch Chrome consistently across operating systems. I found that the Lighthouse project (https://github.com/GoogleChrome/lighthouse/tree/master/chrome-launcher) has isolated their `chrome-launcher` and we'll probably need to use that. I think it should be fairly easy to use their utility to determine if Chrome isn't installed and send that metric.